Reindex case views

dannyroberts opened the pull request:
Depends on:
- [x] https://github.com/dimagi/commcare-hq/pull/7174
- [x] https://github.com/dimagi/commcare-hq/pull/7169

dannyroberts committed:
remove case/by_owner and case/by_owner_lite

dimagimon commented:
Could not review pull request. It may be too large, or contain no reviewable changes.

dannyroberts commented:
I'll probably make a few more changes deleting or changing views in this design doc; I'll add them here and eventually merge them all together for one big reindex

czue commented:
whee

dannyroberts committed:
simplify case/all_cases

and rename to case/open_cases

dimagimon commented on corehq/apps/casegroups/models.py:
```diff
@@ -0,0 +1,46 @@
+    timezone = StringProperty()
+
+    def get_time_zone(self):
+        # Necessary for the CommCareCaseGroup to interact with CommConnect, as if using the CommCareMobileContactMixin
```
E501 line too long (118 > 115 characters)

dimagimon commented on corehq/ex-submodules/casexml/apps/case/models.py:
```diff
@@ -1025,3 +954,7 @@ def _type_sort(action_type):
+
+
+# import signals
+import casexml.apps.case.signals
```
F401 'casexml' imported but unused

dimagimon commented on corehq/ex-submodules/casexml/apps/case/_design/views/open_cases/map.js:
```diff
@@ -0,0 +1,13 @@
+function(doc) {
```
Missing name in function declaration.

dannyroberts committed:
move case/groups_by_domain to casegroups/

dannyroberts committed:
Merge branch 'casegroups' into HEAD

dannyroberts committed:
Merge branch 'get_all_case_owner_ids' into HEAD

dannyroberts committed:
Merge branch 'master' into remove-case-by_owner

benrudolph commented:
:white_check_mark: after tests

dannyroberts commented:
@czue FYI self-merging and starting preindex. Removed 2 'case' views, simplified another one to emit 1/4 the things, and moved another to a separate app.

czue commented:
awesome!
